addons/mail/static/src/model/model_manager.js

```diff
@@ -554,9 +554,6 @@ export class ModelManager {
                 case 'and':
                     break;
                 case 'xor':
```

Comment:
note: switch syntax becomes really useless after this diff

also don't need a separate commit just for this